Tools/autotest/arducopter.py

```diff
@@ -1298,6 +1299,55 @@ def fly_alt_max_fence_test(self):
 
         self.zero_throttle()
 
+    # fly_alt_min_fence_test - fly down until you hit the fence
+    def fly_alt_min_fence_test(self):
+        self.takeoff(50, mode="LOITER", timeout=120)
+        """Hold loiter position."""
+        self.mavproxy.send('switch 5\n')  # loiter mode
+        self.wait_mode('LOITER')
+
+        # enable fence, disable avoidance
+        self.set_parameter("FENCE_ENABLE", 1)
+        self.set_parameter("FENCE_AUTOENABLE", 1)
+        self.set_parameter("AVOID_ENABLE", 0)
+        self.set_parameter("FENCE_TYPE", 8)
+        self.set_parameter("FENCE_ALT_MIN", 20)
+
+        self.change_alt(50)
+
+        # Activate the floor fence
+        # TODO this test should run without requiring this
+        self.run_cmd(mavutil.mavlink.MAV_CMD_DO_FENCE_ENABLE,
```

Comment:
This is needed because copter is not performing an auto-mission. This is handled in the same way as plane. I've pulled the common fence enable functions up to `common.py` to clean this up.

See https://github.com/ArduPilot/ardupilot/pull/15288